.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,173 @@
+# Copyright 2020 New Relic Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: run-integration-tests
+run-name: Testing ${{ inputs.origin }}:${{ inputs.ref }} (by @${{ github.actor }})
+
+on:
+  workflow_dispatch:
+    inputs:
+      origin:
+        description: 'newrelic-php-agent origin'
+        required: true
+        default: 'newrelic' 
+        type: string
+      ref:
+        description: 'Branch or tag'
+        required: true
+        default: 'dev' 
+        type: string
+
+jobs:
+  build-test-runner:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: newrelic/nr-php-agent-builder
+      IMAGE_TAG: make-go
+      IMAGE_VERSION: ${{vars.MAKE_GO_VERSION}}
+    strategy:
+      matrix:
+        platform: [gnu, musl]
+        arch: [amd64, arm64]
+    steps:
+      - name: Checkout newrelic-php-agent code
+        uses: actions/checkout@v3
+        with:
+          path: php-agent
+      - name: Enable arm64 emulation
+        if: ${{ matrix.arch == 'arm64' }}
+        uses: docker/setup-qemu-action@v2
+        with:
+          image: tonistiigi/binfmt:${{vars.BINFMT_IMAGE_VERSION}}
+          platforms: arm64
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build integration_runner
+        run: >
+          docker run --rm --platform linux/${{matrix.arch}}
+          -v "${GITHUB_WORKSPACE}/php-agent":"/usr/local/src/newrelic-php-agent" 
+          -e ACCOUNT_supportability=${{secrets.ACCOUNT_SUPPORTABILITY}}
+          -e APP_supportability=${{secrets.APP_SUPPORTABILITY}}
+          $IMAGE_NAME:$IMAGE_TAG-${{ matrix.platform }}-$IMAGE_VERSION bin/integration_runner
```

Comment:
Below 'make' is part of the docker run command building axiom and the agent, and here it isn't - do these all either need 'make' or not?

Reply:
Also looking for clarification on this

Reply:
`make-go` variant of `newrelic/nr-php-agent-builder` and `make-php` variant of the same image have different entry points. The `make-go` entry point is make. The `make-php` entry point is `docker-php-entrypoint`:
```
$ docker inspect -f '{{.Config.Entrypoint}}' newrelic/nr-php-agent-builder:make-go-gnu-v1
[make -r]
$ docker inspect -f '{{.Config.Entrypoint}}' newrelic/nr-php-agent-builder:make-php-8.2-gnu-v1
[docker-php-entrypoint]
```

Yes, this is a bit unfortunate.